peachpy/x86_64/operand.py

```diff
@@ -427,6 +427,17 @@ def __add__(self, extra_offset):
     def __sub__(self, extra_offset):
         return RIPRelativeOffset(self.offset - extra_offset)
 
+    def __str__(self):
+        return self.format("nasm")
```

Comment:
The default format should be `"peachpy"`

Reply:
Fixed in 764a63e7952e0f5aed151f66d136e159c5928937.